src/routes.ts

```diff
@@ -25,31 +25,63 @@ export default [
     name: ANNUAL_REPORT,
     component: AnnualReport,
     meta: {
-      requiresAuth: true
+      requiresAuth: true,
+      breadcrumb: [
+        {
+          text: 'File Annual Report',
```

Comment:
The mockup shows "File 2018 Annual Report" but I think design won't allow that. Check with Scott or a PO for an exemption :)

Reply:
Will do!

Reply:
I'd like to make this work actually, working on it!

Reply:
Fixed!